### PR TITLE
com.github.jorchube.monitorets

### PR DESCRIPTION
**Pull request by jorchube**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.jorchube.monitorets.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.github.jorchube.monitorets",
```
Comment (hfiguiere) [inline com.github.jorchube.monitorets.json:2]: It should be `io.github` not `com.github`

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.github.jorchube.monitorets

File: com.github.jorchube.monitorets.json (hunk 1 above)
Comment (jorchube) [reply com.github.jorchube.monitorets.json]: I've changed it. Thanks for pointing it out.

Just out of curiosity: I was under the impression that using the `io.github.bla.bla` reverse dns should mean that I have setup a github page for the app.

Is it expected that it exists given the app id?

Thanks again for your help.

**Comment by jorchube** [comment]
bot, build com.github.jorchube.monitorets

**Comment by jorchube** [comment]
bot, build io.github.jorchube.monitorets

File: com.github.jorchube.monitorets.json (hunk 1 above)
Comment (hfiguiere) [reply com.github.jorchube.monitorets.json]: no on github it's defacto yours `io.github.`USERNAME. It's find if it is not setup. It's just namespace ownership.

**Comment by jorchube** [comment]
bot, build io.github.jorchube.monitorets

**Pull request closed without merging**